src/main/resources/templates/partials/header.html

```diff
@@ -28,7 +28,7 @@
                         </a>
                     </li>
                     <li class="govuk-header__navigation-item" th:classappend="${pageCategory == 'actions'} ? 'govuk-header__navigation-item--active'">
-                        <a class="govuk-header__link" href="/notifications">
+                        <a class="govuk-header__link" href="/notifications?notification_type=all">
```

Comment:
could 'all' be the default set by the controller, then no need to specify it here?